cmake/UserOverride.cmake

```diff
@@ -8,30 +8,19 @@
 
 if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     # g++
-    set(common "-std=c++11 -Wall -Wextra -fno-common")
-    if (NOT MINGW)
-        set(common "${common} -fPIC")
-    else()
-        set(common "${common} -D_hypot=hypot -Doff64_t=_off64_t")
-        if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8")
-            set(common "${common} -DMS_WIN64")
-        endif()
-    endif()
-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -ffast-math -funroll-loops")
-    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g")
+    set(common "-Wall -Wextra -Wno-unused-parameter -fno-common")
+    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -march=native -ffast-math -funroll-loops")
+    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -ggdb")
 elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
     # icpc
-    set(common "-std=c++11 -Wall -fno-common")
+    set(common "-Wall -fno-common")
     set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -xHOST -O3 -no-prec-div")
     set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -O0")
 elseif (CMAKE_CXX_COMPILER_ID MATCHES Clang)
     # clang
-    set(common "-std=c++11 -Wall -Wextra -fPIC")
-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -ffast-math -funroll-loops")
-    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g")
+    set(common "-Wall -Wextra -Wno-unused-parameter -fPIC")
```

Comment:
@certik, can you move the fPIC here to CMakeLists.txt?

Reply:
Fixed in 8db6998eb572bbde00602a3c5d11c7da7b84bcee.